docs/pages/access-controls/guides/device-trust.mdx

```diff
@@ -206,17 +208,46 @@ Device trust has the following modes of operation, represented by the
   Additionally, it requires a trusted device for all SSH, Database and
   Kubernetes connections.
 
-Enterprise clusters run in `optional` mode by default. Changing the mode to
-`required` will enforce a trusted device for all SSH, Database and Kubernetes
-accesses.
-
 <Admonition type="warning" title="Web UI">
-The Web UI is not capable of trusted device access during the device trust
-preview. Only `tsh` and Teleport Connect are able to fulfill device mode
-`required`.
+  The Web UI is not capable of trusted device access during the device trust
+  preview. Only `tsh` and Teleport Connect are able to fulfill device mode
+  `required`.
+</Admonition>
+
+### Per-Role
+
+To enforce authenticated device checks for a specific role, update the role as the following:
+
+```diff
+kind: role
+version: v6
+metadata:
+  name: example-role
+spec:
+  options:
+    # require authenticated device check for this role
++   device_trust_mode: "required" # add this line
+  allow:
+    ...
+  deny:
+    ...
+
+```
+
+<Admonition type="tip" title="Per-role vs. Cluster-wide">
+Per-role configuration let's progressively block unauthorized device access and is useful for getting started with Teleport Device Trust.
+
+Once all the devices used by engineers are enrolled to Teleport, cluster-wide configuration will ensure every resources can only ever be accessed with a trusted device.
+
```

Comment:
It's valid but not recommended security wise. Cluster-wide configuration is a failsafe mode, even if there's role misconfiguration, one can be sure that device authentication would be applied.

It's similar case if you compare with 2FA. Having 2FA enabled by default for all is better than conditional 2FA. Of course, it may not be feasible for every use cases and this is where granular policies step in to create exceptional cases. Otherwise, we should advocate for globally applied rules.